build/templates/_converters.py.mako

```diff
@@ -340,11 +340,14 @@ def ${enums[enum_name]['enum_to_converted_value_function_name']}(value):
 
 
 def ${enums[enum_name]['converted_value_to_enum_function_name']}(value):
-    return {
+    try:
```

Comment:
In #1763 I make the case for reverting this.

Reply:
I have addressed this in https://github.com/ni/nimi-python/pull/1763#discussion_r907216778 and cherry-picked the commit to this PR.